admin/sql/updates/2021-06-25-add-optional-mbid-to-pinned-recording-table.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
+
+-- this table is currently empty, we can drop it and recreate it with the new changes
+
+DROP TABLE IF EXISTS pinned_recording;
```

Comment:
I see your logic here -- but I would really prefer to see this redone using ALTER TABLE rather than DROP TABLE. This is more proof against future disasters -- imagine if someone fat-fingers a script name and wants to run a simple update script, but instead gets this script instead by accident. BAM, all of our pins are gone. If this is ALTER TABLE, then the statement will fail (since the column will already exist in this case) and nothing bad happens.

Reply:
I didn't think about this, it makes sense to be extra cautious about using DROP TABLE. :o Done.